### ISSUE TITLE
Re-Embed shortcut produces erroneous prompt in beta.

### ISSUE THREAD

**Issue by bbecausereasonss** [brianpetro/obsidian-smart-connections#631]
Asking to re-embed a note once it already has am embedding produces a dialogue that asks me to re-embed 100% of all my notes. Upon clicking yes, it only re-does the current note. (as it should).

The dialogue is erroneous as it makes it seem like it's going to re-do all of them.

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/20264016/fed7b598-6477-4796-b4ca-f40edcf987a8)

**Comment by uva2212** [comment]
Same issue here. The latest update broke the plugin for me. Continuously processing embeddings. Finishes and then does them again. Stuck in this loop.

**Comment by brianpetro** [comment]
@bbecausereasonss I'm still trying to reproduce the above issue. Are you using the "Refresh & Make Connections" command? 

@uva2212 I am aware of a bug introduced in `2.1.67` that causes a subset of notes to be re-embedded after restarting Obsidian/Smart Connections. Depending on how notes are formatted, this will impact vaults to varying degrees. I anticipate having a fix for this shipped as early as tonight. However, this is different than what @bbecausereasonss is referring to. If the issue still happens after `v2.1.68` (when it's released), please create a new issue, preferably with screenshots of any errors you see in the developer console log so I can further investigate.

🌴

**Comment by uva2212** [comment]
@brianpetro You are right, I raised a separate issue than what @bbecausereasonss raised. That's because it happened with me twice. In fact I canceled on this prompt and it deleted the same amount of notes from my vault altogether. Thankfully I recovered them from sync. I believed that they were connected somehow, so mentioned it here. 

Thank you for your work, looking forward to the fix.

**Comment by brianpetro** [comment]
@uva2212 the update with the fix was just shipped this morning 😊
